src/components/Tables/Tables.js

```diff
@@ -8,16 +8,22 @@ const Tables = ({
   matsTableFlag,
   systemsItems,
 }) => {
+  // console.log('this is section in tables',methodItems)
```

Comment:
i think we should take out commented out code unless it's really needed for something